desktop-widgets/diveplanner.cpp

```diff
@@ -447,7 +447,8 @@ PlannerSettingsWidget::PlannerSettingsWidget(QWidget *parent, Qt::WindowFlags f)
 	disableDecoElements((int) prefs.planner_deco_mode);
 
 	// should be the same order as in dive_comp_type!
-	rebreather_modes << tr("Open circuit") << tr("CCR") << tr("pSCR");
+	for (int i=0; i < FREEDIVE; i++)
+		rebreather_modes.append(QString(divemode_text_ui[i]));
```

Comment:
This imho this removes translation instead of adding it. `divemode_text_ui[i]` holds untranslated strings.